frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -85,8 +115,12 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
                 <RowInstanceHeader
                   viewMode={viewMode}
                   handleSorting={handleSorting!}
+                  showCheckbox={showCheckbox!}
```

Comment:
Avoid skipping null checks, possibly change the prop definition

Reply:
a solution like `showCheckbox || false` could be ok?

Reply:
Yes, actually I just noticed it's a boolean, probably !!showCheckbox is even better. The other props which aren't booleans are more problematic actually but it's out of scope now I guess

Reply:
Ok, thx